GTBitsOfGood/access-h2o
# ci/cd set up

## PR Title/Tagline

sets up cd and fixes linter

### How to Test

1. check the comments for vercel deployments, but need to add env variables before complete

1. `yarn lint` works

### Important Changes

- Database change / migration to run
- Environment config change
- Breaking API change

### Related Github Issues

- #[Number of Issue]
- Or, remember to link to an issue on the sidebar

### Related PRs

- #PRNUMBER

### Checklist
